incubator/wordpress/README.md

```diff
@@ -0,0 +1,111 @@
+| `smtpUser`                           | SMTP user                                | `nil`                                                      |
+| `smtpPassword`                       | SMTP password                            | `nil`                                                      |
+| `smtpUsername`                       | User name for SMTP emails                | `nil`                                                      |
+| `smtpProtocol`                       | SMTP protocol [`tls`, `ssl`]             | `nil`                                                      |
+| `mariadb.mariadbRootPassword`        | MariaDB admin password                   | `nil`                                                      |
+| `serviceType`                        | Kubernetes Service type                  | `LoadBalancer`                                             |
+| `persistence.enabled`                | Enable persistence using PVC             | `true`                                                     |
+| `persistence.apache.storageClass`    | PVC Storage Class for Apache volume      | `generic`                                                  |
+| `persistence.apache.accessMode`      | PVC Access Mode for Apache volume        | `ReadWriteOnce`                                            |
+| `persistence.apache.size`            | PVC Storage Request for Apache volume    | `1Gi`                                                      |
+| `persistence.wordpress.storageClass` | PVC Storage Class for WordPress volume   | `generic`                                                  |
+| `persistence.wordpress.accessMode`   | PVC Access Mode for WordPress volume     | `ReadWriteOnce`                                            |
+| `persistence.wordpress.size`         | PVC Storage Request for WordPress volume | `8Gi`                                                      |
+
+The above parameters map to the env variables defined in [bitnami/wordpress](http://github.com/bitnami/bitnami-docker-wordpress). For more information please refer to the [bitnami/wordpress](http://github.com/bitnami/bitnami-docker-wordpress) image documentation.
+
+Specify each parameter using the `--set key=value[,key=value]` argument to `helm install`. For example,
+
+```bash
+$ helm install --name my-release \
+  --set wordpressUsername=admin,wordpressPassword=password,mariadb.mariadbRootPassword=secretpassword \
+    wordpress-x.x.x.tgz
+```
+
+The above command sets the WordPress application username and password to `admin` and `password` respectively. Additionally it sets the MariaDB `root` user password to `secretpassword`.
+
+Alternatively, a YAML file that specifies the values for the above parameters can be provided while installing the chart. For example,
+
+```bash
+$ helm install --name my-release -f values.yaml wordpress-x.x.x.tgz
```

Comment:
The highlighting with `$ ` vs not having it there.  Looks prettier.  But not a big deal.